Topic6/Exercise3/src/main/java/com/globant/demo/UserController.java

```diff
@@ -0,0 +1,101 @@
+package com.globant.demo;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+import com.globant.models.User;
+import com.globant.services.UserService;
+import com.globant.services.UserServiceImpl;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+@Controller
+public class UserController {
+
+	private UserService userService = new UserServiceImpl();

```

Comment:
This should be using spring boot dependency injection, why instantiate a service manually?